harness/nativeFunctionMatcher.js

```diff
@@ -7,13 +7,16 @@ info: |
     the NativeFunction grammar production without requiring a correct tokeniser.
 
     NativeFunction :
-      function _IdentifierName_ opt ( _FormalParameters_ ) { [ native code ] }
+      function _NativeFunctionAccessor_ opt _IdentifierName_ opt ( _FormalParameters_ ) { [ native code ] }
+    NativeFunctionAccessor :
+      get
+      set
 defines:
   - NATIVE_FUNCTION_RE
   - assertToStringOrNativeFunction
   - assertNativeFunction
 ---*/
-const NATIVE_FUNCTION_RE = /\bfunction\b[\s\S]*\([\s\S]*\)[\s\S]*\{[\s\S]*\[[\s\S]*\bnative\b[\s\S]+\bcode\b[\s\S]*\][\s\S]*\}/;
+const NATIVE_FUNCTION_RE = /\bfunction\b((get|set)\b)?[\s\S]*\([\s\S]*\)[\s\S]*\{[\s\S]*\[[\s\S]*\bnative\b[\s\S]+\bcode\b[\s\S]*\][\s\S]*\}/;
```

Comment:
The `(get|set)` group will never match, since it needs preceding whitespace to be separated from the `function` keyword.

Reply:
that's what the `\b` is, no?

Reply:
The `\b` just means word boundary, but its match length is 0.

Reply:
interesting, i guess this the existing regex technically already works, because it matches way more than it is supposed to. Maybe I'll write a small program that validates this properly.

Reply:
I built something that performs stronger validation, but it fails on the legacy RegExp getter functions (`$&`, `$+`, etc.). Should we use it anyway?